src/core.storage.js

```diff
@@ -35,13 +35,11 @@ class ExtStore {
   }
 
   static create(values, defaults) {
-    return new Promise(async (resolve, reject) => {
-      const store = new ExtStore();
-      for (const key of Object.keys(values)) {
-        await store.setIfNull(values[key], defaults[key]);
-      }
-      resolve(store);
-    })
+    const store = new ExtStore();
+    return Promise.all(Object.keys(values).map((key) => {
```

Comment:
Less ugly version:
```
await Promise.all(...)
return store;
```

Reply:
static method can't be async, so this won't do